aunt-leahs-app/src/App.js

```diff
@@ -2,13 +2,12 @@ import React from 'react';
 import './App.css';
 import AdminLogin from './pages/adminLogin';
 
-
 function App() {
-  return (
-    <div className='testtest'>
-        <AdminLogin />
-    </div>
-  );
+	return (
+		<div className="testtest">
```

Comment:
Did you mean to commit this change? I think that the className could be improved.

Reply:
This is not in the code anymore.